dojo-docs/docs/FAQ.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Frequently Asked Questions
+---
+
+# Dojo / World Modelers FAQ
+
+## 1. Will my model be a good fit for running in Dojo?
+To be a good fit for Dojo your model should be able to run in a Debian/Ubuntu based environment.
+
+It is also useful, but not necessary, to make the most relevant input parameters perturbable before the model run which allows analysts to run “what-if” scenarios with your model.
```

Comment:
This is a note for after this PR (existed before):

There's this article (when running docusaurus locally):

`http://localhost:3001/details/data-format`

-It explains the format that the dataset needs to be in in order to register it. It contains details such as the requirement to possibly manually transpose the dataset data in order for Dojo to process (if the columns/rows are not in what we expect).

I think it might be good to point out that a model output should also follow this convention (Verify?). Mostly pouring my reminder here. We can discuss/assess after merging this PR. Might be useful to point our for model output, if applicable. Maybe not in the FAQ, or maybe a quick link from this FAQ to it (for the model output shape requirement) since it doesn't hurt to have this redundancy.